
Turn 3
================================================================
warning: entity 0 was directed to use 20 halite to move north, but only 0 halite was available
At command 1 of 2:
>>> 1   m 0 n
    2   g


Turn 9
================================================================
warning: entity 0 was directed to use 24 halite to move east, but only 12 halite was available
At command 3 of 5:
    1   m 2 e
    2   m 4 n
>>> 3   m 0 e
    4   m 5 o
    5   g


Turn 12
================================================================
warning: entity 4 was directed to use 13 halite to move north, but only 11 halite was available
At command 5 of 5:
    3   m 5 n
    4   m 2 n
>>> 5   m 4 n


Turn 14
================================================================
warning: entity 5 was directed to use 9 halite to move north, but only 6 halite was available
At command 4 of 5:
    2   m 4 s
    3   m 7 o
>>> 4   m 5 n
    5   m 2 n

warning: owned entities 4, 7 collided on cell (8, 13) as the result of moves on this turn
At command 2 of 5:
    1   m 0 o
>>> 2   m 4 s
    3   m 7 o
    4   m 5 n


Turn 15
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 94, in <module>
    set_long_target(ship)
  File "MyBot.py", line 60, in set_long_target
    if not has_arrival_conflict(ship, cell.position):
  File "MyBot.py", line 26, in has_arrival_conflict
    conflict_ship = me.get_ship(ship_id)
  File "C:\Users\justi\Documents\GitHub\ais-halite\hlt\player.py", line 22, in get_ship
    return self._ships[ship_id]
KeyError: 7

